apps/uptime-kuma/ingressroute.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: traefik.io/v1alpha1
+kind: IngressRoute
+metadata:
+  name: uptime-kuma
+  namespace: default
+spec:
+  entryPoints:
+    - websecure
+    - web
+  routes:
+    - match: Host(`uptime-kuma.pmcd.io`)
```

Comment:
The ingressroute is used for internal access and should remain uptime-kuma.x.pmcd.io